src/main/java/org/codehaus/mojo/license/api/DependenciesTool.java

```diff
@@ -239,4 +264,9 @@
         return result;
     }
     // CHECKSTYLE_ON: MethodLength
+
+    public void setAetherRepoSession( RepositorySystemSession aetherRepoSession )
+    {
+        this.aetherRepoSession = aetherRepoSession;
```

Comment:
Making aetherRepoSession mutable in this way does not feel good. If there is no way to persuade plexus to inject it, I vote for removing the aetherRepoSession field and pass it via method parameter.

Reply:
Replaced with MavenSession injection